sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -542,11 +542,17 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
           )
           removeEmptyObjects(formattedData)
           const result = editMode
-            ? await listingsService.update({
-                listingId: listing.id,
-                body: { id: listing.id, ...formattedData },
-              })
-            : await listingsService.create({ body: formattedData })
+            ? await listingsService.update(
+                {
+                  listingId: listing.id,
+                  body: { id: listing.id, ...formattedData },
+                },
+                { headers: { "x-purge-cache": true } }
+              )
+            : await listingsService.create(
+                { body: formattedData },
+                { headers: { "x-purge-cache": true } }
+              )
```

Comment:
The problem with this is that we're setting the cache keys as $proxy_host$request_uri, so every iteration of listings we call has its own key and we have to call that full key to clear it. So we have to call routes like `/listings`, `/listings/d22911b6-26e2-49d4-bcbb-040fe0f564cb`, `/listings?view=base&limit=all&filter%5B0%5D%5B%24comparison%5D=%3C%3E&filter%5B0%5D%5Bstatus%5D=pending&filter%5B1%5D%5B%24comparison%5D=%3D&filter%5B1%5D%5Bjurisdiction%5D=6ab8534d-870c-4ab3-b6e5-e7dd486ee27a`, etc to clear them.

The good news is that I think we only really need to clear the detail and list routes and we know that list route has these params:
```
params: {
  view: "base",
  limit: "all",
  filter: [
    {
      $comparison: "<>",
      status: "pending",
    },
    {
      $comparison: "=",
      jurisdiction: <jurisdictionId>,
    },
  ],
},
```
Longer term, we will want an easier way to keep track of these keys to use them, especially if we introduce filtering from the public site.

Also, in using this, I couldn't ever get the cache to clear and had to manually call to clear it, so something isn't quite right here as it is. We may have to make separate GET requests to these two routes after the save is successful.